### ISSUE TITLE
bug: parallel space WhatsApp crashing 

### ISSUE THREAD

**Issue by kanishk-upadhyay** [crdroidandroid/issue_tracker#230]
### Issue type

Source code related

### Device

RMX3031

### crDroid version

crDroid 9

### Exact version / Build date

9.8/2023-08-17

### Bug description

app crashes

### Steps to reproduce

opened whatsapp in parallel space

### Relevant log (logcat/build log)

```shell
Generated by AppErrorsTracking
Project Url: https://github.com/KitsunePie/AppErrorsTracking
===============
[Device Brand]: realme
[Device Model]: Realme X7 Max 5G
[Display]: TQ3A.230805.001 release-keys
[Android Version]: 13
[Android API Level]: 33
[System Locale]: en_IN
[Process ID]: 23538
[User Id]: 10
[CPU ABI]: arm64-v8a
[Package Name]: com.whatsapp
[Version Name]: 2.23.16.76
[Version Code]: 231676004
[Error Type]: JVM
[Crash Time]: 2023-08-27T17:13:19.238
[Stack Trace]:
java.lang.IllegalArgumentException: com.whatsapp: Targeting S+ (version 31 and above) requires that one of FLAG_IMMUTABLE or FLAG_MUTABLE be specified when creating a PendingIntent.
Strongly consider using FLAG_IMMUTABLE, only use FLAG_MUTABLE if some functionality depends on the PendingIntent being mutable, e.g. if it needs to be used with inline replies or bubbles.
	at android.app.PendingIntent.checkFlags(PendingIntent.java:401)
	at android.app.PendingIntent.getBroadcastAsUser(PendingIntent.java:671)
	at android.app.PendingIntent.getBroadcast(PendingIntent.java:658)
	at X.2cw.A00(:302515)
	at com.facebook.redex.RunnableRunnableShape0S0300000.run(:590815)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
	at java.lang.Thread.run(Thread.java:1012)
```

### Acknowledgements

- [X] I've checked device is officially supported (for device specific reports and not source related).
- [X] I'm running latest version available on crdroid.net for this device
- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by gwolf2u** [comment]
This is deemed as #wontfix
Whatsapp will soon release ability to add secondary account anyway
